### ISSUE TITLE
remotecall_fetch with lambda function makes error.

### ISSUE THREAD

**Issue by usefulhyun** [JuliaLang/julia#22865]
```
addprocs()
@everywhere function time_remotecall_fetch_to(target, s)
        d = ones(s, s)
        tic()
        remotecall_fetch(x->x.+1, target, d)
        toq()
end
remotecall_fetch(time_remotecall_fetch_to, 2, 3, 100)
```
This code works in 0.6.0-rc2.0
But it is not work in current version.

**Comment by amitmurthy** [comment]
This works for me on master. Which version does it not work on?

**Comment by amitmurthy** [comment]
Can confirm this fails on 0.6, works on master. I believe https://github.com/JuliaLang/julia/pull/22675 also fixed this issue and it is backport pending.

Should be fixed in the next point release of 0.6 .
